src/corona_hakab_model/analyzers/population_analysis.py

```diff
@@ -90,7 +90,7 @@ def export_csv(self, circle_file_path: str, age_file_path: str):
     parser.add_argument("--no-view",dest='view',default=True,action='store_false',help="Skip viewing the generated histograms.")
     parser.add_argument("--no-save",dest='save',default=True,action='store_false',help="Skip saving the resulting CSV")
     args = parser.parse_args()
-    directory = Path(args.directory or project_structure.OUTPUT_FOLDER)
+    directory = Path(args.directory or project_structure.MODEL_FOLDER / "output")
```

Comment:
Why? Isn't it more appropriate to bundle the anlysis of the population with the population data in our output folder?

Reply:
population analysis is not done in the same run as the rest of the simulation,it is done afterwards (maybe we should change that, but this is how it works right now).

OUTPUT_FOLDER was updated to be:
`OUTPUT_FOLDER = MODEL_FOLDER / "output" / output_subdir
which means, it is looking for a subdir for the current run. since the population analysis is done alone, there is no such folder. (it is looking for a folder inside outputs, with a named based on the current time, and since this folder doesn't exists, it crushed).

using MODEL_FOLDER / "output" is the only way to get to the output folder right now. maybe we should change that too.